lib/public/utilities/formatting/formatSizeInBytes.js

```diff
@@ -11,14 +11,17 @@
  * or submit itself to any jurisdiction.
  */
 
-const DATA_SIZE_UNITS = ['B', 'kB', 'MB', 'GB', 'TB'];
+const DATA_SIZE_UNITS = ['B', 'kB', 'MB', 'GB', 'TB', 'PB'];
 
 /**
  * Format size in bytes as human readibly
  * @param {number} size number of bytes to be formatted
  * @return {string} formatted number
  */
 export const formatSizeInBytes = (size) => {
+    if (size === null || size === undefined) {
+        return '-';
+    }
     const exponent = size == 0 ? 0 : Math.floor(Math.log(size) / Math.log(1024));
```

Comment:
Because you have a limited list of data sizes, maybe you should also limit the exponent ? So if we go to exabyte (not sure it can happen, if it can't then forget about it) we display for example `1543` PB and not just `1.5 undefined`